test/functional/tests/incremental_load/test_core_pool.py

```diff
@@ -36,24 +37,32 @@ def test_attach_core_pool():
         core_disk.create_partitions([Size(2, Unit.GibiByte), Size(2, Unit.GibiByte)])
         core_dev = core_disk.partitions[0]
         second_core_dev = core_disk.partitions[1]
+
     with TestRun.step("Start cache."):
         cache = casadm.start_cache(cache_dev, force=True)
+
     with TestRun.step("Add core device."):
-        cache.add_core(core_dev)
+        core = cache.add_core(core_dev)
+
     with TestRun.step("Stop cache."):
         cache.stop()
+
     with TestRun.step("Add previously used core device to core pool using --try-add flag."):
-        first_core = casadm.try_add(core_dev, cache.cache_id)
+        first_core = casadm.try_add(core_dev, cache.cache_id, core.core_id)
```

Comment:
I recently missed that - please left it as it was before, like I said in older comments